doc/Release_Build_Creation.rst

```diff
@@ -4,6 +4,23 @@ Release Build Creation
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
+***************
+### Hardware Specifications - Single-Node VM deployment
+***************
+- Please create VM(s) with at least 1 Node, 2 cores (vCPU) and 4GB of RAM.
+- Ensure the VM is created with minimum 6 attached disks (raw) per node i.e. /dev/sdb (Data Disk), /dev/sdc (Metadata Disks)
+
+  - Data Disks - Min 4 Disks per node (Capacity 10G+)
+  - Metadata Disks - Min 2 Disks per node (Capacity - 10% of total Data Disk Size)
+  
+- Ensure the VM is created with minimum 3 network Interfaces (recommended 5 interfaces)
```

Comment:
Not sure if anyone tested with 3 interfaces and 4+2 Disks.

Reply:
Lets go with 4+2+0 as this is tested so Wiki updated accordingly